components/ai_chat/resources/page/components/context_menu_assistant/index.tsx

```diff
@@ -139,11 +142,19 @@ function ContextMenuAssistant_(
 
   return (
     <>
-      <ButtonMenu className={classnames(styles.moreButton, props.className)}>
+      <ButtonMenu
+        className={classnames({
+          [styles.moreButton]: true,
+          [styles.moreButtonHide]: loadTimeData.getBoolean('isMobile')
```

Comment:
because nala's button-menu component render's a default if there's no anchor-content slot.

<img width="193" alt="image" src="https://github.com/brave/brave-core/assets/8665427/32c385bc-6ed9-4fda-a522-3671b56e365f">